kobo/django/forms.py

```diff
@@ -46,7 +46,8 @@ def clean(self, value):
 class JSONWidget(django.forms.widgets.Textarea):
     def render(self, name, value, attrs=None, renderer=None):
         if value is None: value = ''
-        final_attrs = self.build_attrs(attrs, name=name)
+        final_attrs = self.build_attrs(attrs)
```

Comment:
According to the [sources of Django](https://github.com/django/django/blob/513441240f874dd0b6187c0c6aaa3e8eccd8ddbe/django/forms/widgets.py#L254), it should be possible to provide extra_attrs as the second argument:
```suggestion
        final_attrs = self.build_attrs(attrs, {"name": name})
```
But the proposed solution works as well so feel free to ignore my comment